### PR TITLE
Update preload extension documentation: form preloading and preload="always"

### PR DESCRIPTION
**Pull request by marisst**
Pull request should be viewed together with corresponding PR in htmx-extensions repository https://github.com/bigskysoftware/htmx-extensions/pull/106

Added documentation about
- `HX-Preload` header
- form element preloading
- `preload="always"` configuration

### CODE REVIEW

**Comment by Telroshan** [comment]
Note: leaving this PR open & on the side until the `preload` extension gets published with the latest update (currently `2.0.1` is still the `latest` on npm, thus these new docs are not correct _yet_)

File: www/content/extensions/preload.md [hunk 1]
```diff
@@ -115,6 +126,13 @@ trigger preloads as soon as an object has been processed by htmx.
 </body>
 ```
 
+#### preload="always"
+
+By default, the extension will preload each element once.
+If you would like to keep always preloading the element, you can add `preload="always"` attribute.
+This can be useful if `hx-target` is not the element itself.
+This attribute can be conbined with other configuration attributes e.g. `preload="always mouseover"`.
```
Comment (Telroshan) [inline www/content/extensions/preload.md:134]: Just a little typo then I'll merge it!

```suggestion
This attribute can be combined with other configuration attributes e.g. `preload="always mouseover"`.
```

**Review by Telroshan — CHANGES_REQUESTED** [review]

File: www/content/extensions/preload.md (hunk 1 above)
Comment (marisst) [reply www/content/extensions/preload.md]: Done :)

**Review by Telroshan — APPROVED** [review]
Nice, and thanks again for your work on the extension 😄

**Comment by pvs-stm** [comment]
Is the link shown in https://htmx.org/extensions/preload/ correct?
`<script src="https://cdn.jsdelivr.net/npm/htmx-ext-preload@2.1.0" integrity="sha384-fkzubQiTB69M7XTToqW6tplvxAOJkqPl5JmLAbumV2EacmuJb8xEP9KnJafk/rg8" crossorigin="anonymous"></script>`

It is producing this output in console:
`Failed to find a valid digest in the 'integrity' attribute for resource 'https://cdn.jsdelivr.net/npm/htmx-ext-preload@2.1.0' with computed SHA-384 integrity 'KKYgXDj/WvJAt1FI2JsD8OKSFwQwuHzZWo6EEPm0Yirsk7plaoACUYnv0qlxgOB7'. The resource has been blocked.`

Also, this version doesn't seem to fix the problem of the double download of contents instead of using the cache. I'm still having this exact issue: https://github.com/bigskysoftware/htmx-extensions/issues/108

**Pull request merged**